docs/releases/ez_platform_v3.0_project_update.md

```diff
@@ -102,15 +107,29 @@ class MyService
 }
 ```
 
-## Renamed templates and template parameters
+### User checker
+
+Add the user checker to firewall by adding the following line to `config/packages/security.yaml`:
+
+``` yaml
+security:
+    firewalls:
+        ezpublish_front:
+            # ...
+            user_checker: eZ\Publish\Core\MVC\Symfony\Security\UserChecker
```

Comment:
Looking at this now, do you think we need to add `hl_lines=5` here? The command says about adding a single line while the example has a few of them. :-)